src/lib/transition_frontier_controller_tests/test_sync_handler.ml

```diff
@@ -16,7 +16,7 @@ open Stubs
 
 let%test_module "Sync_handler" =
   ( module struct
-    let logger = Logger.null ()
+    let logger = Logger.create ()
```

Comment:
I believe we intentionally turned these logs off. Should we use a different logger for the heartbeat?